### ISSUE TITLE
Link preview not available on Mastodon

### ISSUE THREAD

**Issue by tonydiep** [Automattic/wordpress-activitypub#403]
I'm not sure this plugin is using the custom post-content.

```
<p><strong>[ap_title]</strong></p>
<div>
[ap_image]
</div>
<p>[ap_permalink]</p>
```

![Screenshot from 2023-08-22 12-43-06](https://github.com/Automattic/wordpress-activitypub/assets/705269/4697f960-114a-45aa-bdc8-d6da1c991fae)

Mastodon shows "Preview not available" and the content does not match what I specified.

![Screenshot from 2023-08-22 12-28-49](https://github.com/Automattic/wordpress-activitypub/assets/705269/d75f8b80-f7e5-4bf4-ab68-e468a4bfb938)

**Comment by pfefferle** [comment]
Do you use "Note" as Post-Type? If not, please change the settings to note, otherwise there is no guarantee that Mastodon will show what you expect. The link preview is not part of the ActivityPub spec, to add support for link previews, you have to install an OpenGraph plugin.

**Comment by tonydiep** [comment]
Thanks for replying.  I will try those things.

- Tony Diep (He/Him)
________________________________

**Comment by pfefferle** [comment]
Let me know if it works!

**Comment by timnolte** [comment]
@tonydiep the setting that you are supposed to check/change is in the ActivityPub plugin settings.


Activity-Object-Type
 Note (default) - Should work with most platforms.

 Article - The presentation of the "Article" might change on different platforms. Mastodon for example shows the "Article" type as a simple link.

 WordPress Post-Format - Maps the WordPress Post-Format to the ActivityPub Object Type.

**Comment by tonydiep** [comment]
Hi Tim,

Thank you for pointing me to the right setting.

It is set to "Note"

- Tony Diep (He/Him)
________________________________

**Comment by tonydiep** [comment]
Updated Activity Pub from 0.17 to 1.0 today and now my photo posts show up on Mastodon with previews!  Thank you so much for making this plugin!

(I previously migrated from wordpress.com to my own server just so I could run this plugin. Today was the day I migrated back because the previews weren't showing up. So happy to see the previews are working!)

![image](https://github.com/Automattic/wordpress-activitypub/assets/705269/ebe9e56f-c403-415d-af34-68941b62cfec)
